behavior-modules/greet.js

```diff
@@ -0,0 +1,7 @@
+const Behavior = require('../models/behavior');
```

Comment:
yes. I did do that first. but then `behaviors.js` and `behaviors` directory point to the same module. Perhaps we should rename the folder to `behaviors` and move `behaviors.js` inside as `behaviors/index.js`? I mentioned this in the PR body too :)